package.json

```diff
@@ -33,6 +33,9 @@
     "dist/assets/styles/configuration.css"
   ],
   "dependencies": {
+    "@lit-labs/motion": "^1.0.7",
```

Comment:
I think these dependencies are devDependencies (as it will be bundled)

Reply:
(all 4 dependencies, including rxjs)

Reply:
Weird, I thought I asked it to put them in dev. Let me fix that